### DISCUSSION TITLE
Have to restart vscode after `prisma generate`

### DISCUSSION THREAD

**Discussion by kevinvdburgt** [prisma/prisma#3474] [Q&A] [answered]
I have to restart ascode after running `prisma generate` in order to use autocompletion (prisma extension is installed). Is there a way to reload without restarting vscode after each `prisma generate`?

**Comment by ryands17** [comment, marked as answer]
@kevinvdburgt Yes this is a known issue and what you could do for now is press `ctrl+shift+p` or `cmd+shift+p` if on a mac to open the VSCode operations toolbar and type this: Restart TS Server

This should load all the updated types.

![image](https://user-images.githubusercontent.com/19697099/91822380-57f0de00-ec55-11ea-8714-063679651f32.png)

**Reply by janacm** [reply]
Thanks for the fix - do you happen to know if there is an existing link to this issue?

**Reply by joaopedrodcf** [reply]
I'm pretty sure is this issue here @janacm https://github.com/prisma/prisma/issues/14722

**Comment by ibrahimyaacob92** [comment]
I wish this is somewhat automated

**Reply by kevinvdburgt** [reply]
I haven’t had this issue for a long time.

Are you sure you’re on the latest versions?

**Reply by victorhsalves** [reply]
I'm using prisma 4.5.0 and this problem still happening.

**Reply by austincrim** [reply]
This issue is more relevant to VS Code and the TypeScript Language Server. Feel free to open an issue on [their repo](https://github.com/microsoft/vscode).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
